src/client/Microsoft.Identity.Client/Internal/Requests/SilentRequest.cs

```diff
@@ -20,6 +20,7 @@ internal class SilentRequest : RequestBase
     {
         private readonly AcquireTokenSilentParameters _silentParameters;
         private const string TheOnlyFamilyId = "1";
+        private bool? _canExecuteBrokerValue;
```

Comment:
There is a better way to do lazy initialization using `Lazy<T>`. You can see an example here: https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/b84322c945e0b8e5f3e237ea9b9e695b50fd1a36/src/client/Microsoft.Identity.Client/Cache/Items/MsalIdTokenCacheItem.cs#L19

While I'm normally not bothered by doing things "manually", I think using Lazy<T> would make the code more readable.

Reply:
Would have been nice to address this as suggested.